pom.xml

```diff
@@ -285,87 +285,5 @@
 				</plugins>
 			</build>
 		</profile>
-		
```

Comment:
I can't accept this into master yet.

Reply:
It's odd that Github is flagging that as I copy/pasted the original pom.xml from the master branch, but I was able to update the file so that only whitespace differences are still marked. Let me know if that's acceptable, thanks!